crates/bevy_pbr/src/ssao/mod.rs

```diff
@@ -114,9 +114,14 @@ impl Plugin for ScreenSpaceAmbientOcclusionPlugin {
             .init_resource::<SsaoPipelines>()
             .init_resource::<SpecializedComputePipelines<SsaoPipelines>>()
             .add_systems(ExtractSchedule, extract_ssao_settings)
-            .add_systems(Render, prepare_ssao_textures.in_set(RenderSet::Prepare))
-            .add_systems(Render, prepare_ssao_pipelines.in_set(RenderSet::Prepare))
-            .add_systems(Render, queue_ssao_bind_groups.in_set(RenderSet::Queue))
+            .add_systems(
+                Render,
+                (
+                    prepare_ssao_pipelines.in_set(RenderSet::Prepare),
```

Comment:
This (and the other pipelines) are just in `Prepare` because they don't have any dependencies (before or after) with any other `Prepare` systems. All the ones in `PrepareResource` are in there because something in `PrepareBindgroups` relies on them being done first.